### PR TITLE
New datasets schemas for malformed and not informed

### PR DESCRIPTION
**Pull request by mercesanchez**
New samples for malformed datasets

### CODE REVIEW

File: samples/datasets/qa_not_informed_fields_logical_types/v1.0.0/qa_not_informed_fields_logical_types_1.0.0.avsc [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+                },
+                "com.telefonica.NOT_INFORMED"
+            ],
+            "doc": "A valid ISO-3166-1 alpha-2 country code"
+        },
+        {
+            "name": "COUNTRY_CODE3",
+            "type": [
+                {
+                    "type": "string",&& $"DATE.member0".isNotNull
```
Comment (pradomota) [inline samples/datasets/qa_not_informed_fields_logical_types/v1.0.0/qa_not_informed_fields_logical_types_1.0.0.avsc:130]:
```suggestion
                    "type": "string",
```

File: samples/extensions/qa_malformed_not_informed/v1.0.0/extensions.json [hunk 2]
```diff
@@ -0,0 +1,171 @@
+{
+    "type": "record",
+    "name": "qa_malformed_not_informed",
```
Comment (pradomota) [inline samples/extensions/qa_malformed_not_informed/v1.0.0/extensions.json:3]: we can use the new malformed-rate attr for malformed data.

File: samples/extensions/qa_not_informed_fields_logical_types/v2.0.0/extensions.json [hunk 3]
```diff
@@ -0,0 +1,51 @@
+{
+    "type": "record",
+    "name": "qa_not_informed_fields_logical_types",
+    "namespace": "com.telefonica.baikal.qa",
```
Comment (pradomota) [inline samples/extensions/qa_not_informed_fields_logical_types/v2.0.0/extensions.json:4]: This extension file is not necessary, just remove this folder

Comment (mercesanchez) [reply samples/extensions/qa_not_informed_fields_logical_types/v2.0.0/extensions.json:4]: OK

**Review by pradomota — APPROVED** [review]

**Pull request merged**